ddtrace/internal/remoteconfig/worker.py

```diff
@@ -15,7 +16,18 @@
 
 def get_poll_interval_seconds():
     # type:() -> float
-    return float(os.getenv("DD_REMOTECONFIG_POLL_SECONDS", default=DEFAULT_REMOTECONFIG_POLL_SECONDS))
+    if os.getenv("DD_REMOTECONFIG_POLL_SECONDS"):
+        deprecate(
+            "Using environment variable 'DD_REMOTECONFIG_POLL_SECONDS' is deprecated",
+            message="Please use DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS instead.",
+            removal_version="2.0.0",
+        )
```

Comment:
Do you reckon we need this deprecation notice? My understanding was that this was effectively a private variable that we did not document.

Reply:
It going to be a public variable now and this variable is on 1.7 release, I think the message could be useful to be sure anyone  implements `DD_REMOTECONFIG_POLL_SECONDS`. On the other hand, you're right, it's not documented, i'm going to create a new PR with that)